source/server/publishers/authentication-publisher.js

```diff
@@ -0,0 +1,32 @@
+Space.Object.extend(Space.accounts, 'AuthenticationPublisher', {
+
+  mixin: [
+    Space.messaging.EventPublishing
+  ],
+
+  dependencies: {
+    accounts: 'Accounts'
+  },
+
+  onDependenciesReady() {
+    this.accounts.onLogin(_.bind(this.handleSuccessfulLogin, this));
+    this.accounts.onLoginFailure(_.bind(this.handleFailedLogin, this));
+  },
+
+  handleFailedLogin(login) {
```

Comment:
Maybe just move this into the onDependenciesReady block. No point in complicating things, and the testing will be based on event being published rather than method being called.